### DISCUSSION TITLE
Cannot connect to ASReview Lab from device outside of hosting device

### DISCUSSION THREAD

**Discussion by danielzx2** [asreview/asreview#1680] [Troubleshooting (report errors in the "Issues" tracker)]
I want to access the instance of ASReview Lab that is hosted on a server in my local network. I type in the public ip address, port 5000 IP and "/projects" for the path on another computer, and it refuses to connect. I get error "ERR_CONNECTION_REFUSED". I checked my firewalls, allowing for port 5000 to pass through but nothing changed. What do I do?

Server Setup:
OS: Ubuntu 22.04
Python Version: 3.10.12

**Comment by cskaandorp** [comment]
Hi Daniel,

So you go to http://xx.xx.xx.xx:5000/projects and you get a connection error. 

There are many things that can go wrong. First question that I have is how you have started the application. Did you use a configuration toml file? Or just fired up the server? By default the app is now using authentication, which means you have to signup/signin if you want to go to /projects. Another obstacle might be CORS... is your public ip part of the ALLOWED_ORIGINS list of the backend. Do you use Nginx in front of the application?

These are just a couple of questions, and maybe these questions are too simple (maybe you have thought this whole thing through). If you want, we can both have a look via Teams or Zoom.

cheers

Cspr
